http/lang/en/farmer.lang

```diff
@@ -85,5 +86,6 @@ victories			victories
 warnings			advertisements
 warning_severity_s			Severity %s
 wins			wins
+you_must_have_2_leeks		You need at least 2 leeks to register at a farmer tournament
```

Comment:
J'aurais utilisé `register for the` au lieu de `register at a`, mais c'est un peu les goûts les couleurs